File: JAVA PROGRAMMING/Test13.java
// 13. PRINT WHETHER THE NUMBER IS EVEN OR ODD
import java.util.Scanner;
public class Test13 {
    public static void main(String args[]) {
        Scanner in = new Scanner(System.in);

        System.out.println("Enter Input Value: ");
        int input = in.nextInt();

        String res = (input % 2 == 0) ? "Even Number" : "Odd Number";
        System.out.println(input+" Is "+res);
        in.close();
    }
}

File: JAVASCRIPT PROGRAMMING/Test13.js
"use strict";
// 13. PRINT WHETHER THE NUMBER IS EVEN OR ODD
const func = (ip) => {
    if (ip % 2 == 0) {
        console.log(`${ip} Is Even Number.`);
    }
    else {
        console.log(`${ip} Is Odd Number.`);
    }
};
func(12);
func(-14);
func(0);
func(19);
func(-23);
func(0);